staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -268,22 +331,33 @@ func (_ networkInterface) Interfaces() ([]net.Interface, error) {
 	return net.Interfaces()
 }
 
-func chooseHostInterfaceFromRoute(inFile io.Reader, nw networkInterfacer) (net.IP, error) {
-	routes, err := getRoutes(inFile)
+// getAllDefaultRoutes obtains IPv4 and IPv6 default routes on the node. If unable
```

Comment:
What happens if we can't get the IPv4 default routes, but can get IPv6 routes? Is that an error? Should it be an error?

I'm making this comment from reading just the docstring